app/auth.py

```diff
@@ -49,16 +41,12 @@ def login():
     form = LoginForm()
     if request.method == "POST":
         # Check if username already exists in db
-        user = mongo.db.users.find_one(
-            {"username": request.form.get("username").lower()})
-  
+        user = user = User.find_one_user(request.form.get("username"))
```

Comment:
Hi @stefdworschak `user = user = User...` is there an extra `user = ` assignment here that doesn't need to be here?

Reply:
Nope. No idea what happened there :rofl:! Removed it now.